src/Umbraco.Web.UI.Client/src/views/propertyeditors/slider/slider.controller.js

```diff
@@ -122,7 +212,7 @@
         });
 
     //load the separate css for the editor to avoid it blocking our js loading
-    assetsService.loadCss("lib/slider/slider.css");
-
+    assetsService.loadCss("lib/slider/bootstrap-slider.css");
+    assetsService.loadCss("lib/slider/bootstrap-slider-custom.css");

```

Comment:
Not sure if we could load these once instead of two requests? because it might take about half an second before the custom style overwrite the default style on init.
Furthermore the bootstrap-slider.css also hide the input field, but because it wait with loading the css you might also see the input field on init.

Maybe have something like ```display:none``` on the element until it is ready?